docs/DeveloperGuide.md

```diff
@@ -20,18 +197,18 @@
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ... | I want to ...             | So that I can ...                                           |
+| ------- | -------- | ------------------------- | ----------------------------------------------------------- |
+| v1.0    | new user | see usage instructions    | refer to them when I forget how to use the application      |
+| v2.0    | user     | find a to-do item by name | locate a to-do without having to go through the entire list |
 
 ## Non-Functional Requirements
```

Comment:
Perhaps you could elaborate on the non-functional requirements that your group is trying to achieve in your application, which may be linked to your user stories.